client/src/components/Dropdown/Dropdown.js

```diff
@@ -23,13 +23,13 @@ class Dropdown extends Component {
     let button = [];
     switch (type) {
       case 'icon':
-        button.push(<DropdownIcon icon={this.props.icon} />);
+        button.push(<DropdownIcon key="icon" icon={this.props.icon} />);
```

Comment:
this, and the other change in this file, are to stop the JavaScript error that appears asking for `key` to be provided in this component. The change is just to eliminate the error, this doesn't change any functionality.